### PR TITLE
feature: add name require for merged image

### PR DESCRIPTION
**Pull request by mahmut-Abi**
#1026 
### Describe what this PR does / why we need it
add name require for merged image

### CODE REVIEW

File: sealer/cmd/merge.go [hunk 1]
```diff
@@ -53,5 +54,10 @@ merge images:
 
 func init() {
 	rootCmd.AddCommand(mergeCmd)
-	mergeCmd.Flags().StringVarP(&ImageName, "image-name", "t", "", "target image name")
+	mergeCmd.Flags().StringVarP(&ImageName, "merged-imageName", "t", "", "target image name")
```
Comment (fanux) [inline sealer/cmd/merge.go:57]: merged-imageName -> target-image

Comment (mahmut-Abi) [reply sealer/cmd/merge.go]: OK

File: sealer/cmd/merge.go [hunk 2]
```diff
@@ -53,5 +57,10 @@ merge images:
 
 func init() {
 	rootCmd.AddCommand(mergeCmd)
-	mergeCmd.Flags().StringVarP(&ImageName, "image-name", "t", "", "target image name")
+	mergeCmd.Flags().StringVarP(&ImageName, "target-image", "t", "", "target image name")
+
+	if err := mergeCmd.MarkFlagRequired("target-image"); err != nil {
```
Comment (fanux) [inline sealer/cmd/merge.go:62]: Put those check into RunE function, then return error if failed.

File: sealer/cmd/merge.go [hunk 3]
```diff
@@ -43,6 +44,9 @@ merge images:
 			}
 			images = append(images, image)
 		}
+		if !strings.Contains(ImageName, ":") {
+			ImageName = ImageName + ":latest"
```
Comment (kakaZhou719) [inline sealer/cmd/merge.go:48]:
```
func buildRaw(name string) string {
	i := strings.LastIndexByte(name, ':')
	if i == -1 {
		return name + ":" + defaultTag
	}
	if i > strings.LastIndexByte(name, '/') {
		return name
	}
	return name + ":" + defaultTag
}
```

**Pull request closed without merging**